test/api-test.js

```diff
@@ -50,6 +50,33 @@ describe('RippleAPI', function() {
       _.partial(checkResult, responses.preparePayment, done));
   });
 
+  it('preparePayment - 2', function(done) {
```

Comment:
Yeah, I can try..
But here it will be something like 'preparePayment with lot of flags and options'...